dali/python/nvidia/dali/plugin/triton.py

```diff
@@ -29,7 +29,7 @@ def autoserialize(dali_pipeline):
     For more information about Triton, please refer to [Triton documentation](https://github.com/triton-inference-server/server#triton-inference-server).
 
     :param dali_pipeline: DALI Python model definition (``pipeline_def``).
-    """
+    """  # noqa W505
```

Comment:
How about just breaking the lines, that are too long? This shouldn't be rendering incorrectly after such change.